### ISSUE TITLE
controlnet is not working

### ISSUE THREAD

**Issue by DeahtKalif** [lllyasviel/ControlNet#223]
i am using a M1 pro 10 cpu 16 gpu
i have installed homebrew, opencv-python, automatic1111, downloaded the models
i followed all the instructions of a thousand youtube videos to see if i am missing something
i am trying to use the scribble, but whatever i draw, its not using it... its like controlnet is doing nothing

**Comment by nfarina** [comment]
Same happening for me on M1 Air; scribble got me super excited to set up all this stuff for the first time just to play with it, but my input images (even those copied from other successful examples online) are _mostly_ ignored. Not completely ignored, but mostly. I wish I could explain better, and I am probably doing something dumb, but this is a new world for me.

**Comment by DeahtKalif** [comment]
> Same happening for me on M1 Air; scribble got me super excited to set up all this stuff for the first time just to play with it, but my input images (even those copied from other successful examples online) are _mostly_ ignored. Not completely ignored, but mostly. I wish I could explain better, and I am probably doing something dumb, but this is a new world for me.

yup i hope someone can help us, dont know what we are doing wrong
